.github/resources/pypiserver/base/pypiserver.yaml

```
@@ -0,0 +1,78 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pypi-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: pypi-server
+  template:
+    metadata:
+      labels:
+        app: pypi-server
+    spec:
+      containers:
+        - image: "quay.io/harshad16/pypi-server"
+          imagePullPolicy: IfNotPresent
+          name: pypi-server
+          command:
+            - pypi-server
+          args:
+            - run
+            - '--host'
+            - '0.0.0.0'
+            - '--port'
+            - '8080'
+            - '--disable-fallback'
+            - '--verbose'
+            - '/opt/app-root/packages'
+          ports:
+            - containerPort: 8080
+              protocol: TCP
+          resources:
+            requests:
+              cpu: "0.02m"
```

```suggestion
              cpu: 20m
```
we can afford more here, can you change this to `10m` and all other cpu occurrences to to `10m` as well. 

and let's raise the memory to `100Mi`, 12Mi feels like it will result in really slow downlaods.